content/Blog/2018-02-01-design-the-best-architecture-ever.md

```diff
@@ -0,0 +1,79 @@
+{
+    "title": "How to determine the system architecture?",
+    "date": "2018-02-01"
+}
+
+# SYSTEM ARCHITECTURE
+
+
+## What's an architecture ?
+
+A system can be considered as a group of subsystems with their own functionalities and goals.
+The architecture is both the general structure distribution of sub-modules on board, and the definition of their relations.
+
+We previously presented our 5 modules ADCS, TCS, OBC, EDT and EPS. Our architecture has to create subsystems in terms of hardware, communication and energy, while respecting spatial requirements and being the most failure-tolerant.
+
+----------
+
+## How to find the best architecture ? (methodology)
+For a long and ambitious project like ECE3SAT, we wished to follow embedded system methodology.
+We applied the process found in the book *Embedded Systems Architectures* described in the following article.
+
+#### 1. Having a solid technical base
+Our project covers various technical realms. The first aspect is to gather knowledge in order to understand most requirements.
+
+#### 2. Understanding the architecture business cycle
+First of all, we had to gather all requirements from each module. The previous phase focused on it and used formalism to describe it in their reports. However, some requirements had to be updated, that's why dialog with other modules (our final clients) was crucial.
+
+We kept in mind that any information has to be saved (even if it’s written on a napkin). Indeed, we have to be able to make our choices understandable by future working teams.

```

Comment:
"every information should be somehow logged and saved" ?
"make ~our~ choices that are understandable"